docs: add HDD-supported document (#213)
harvester/docs: docs/advanced/hdd-support.md

---
sidebar_position: 2
sidebar_label: HDD Support
title: ""
keywords:
  - HDD
  - StorageClasses
  - StorageTiering
  - Tiering
  - ColdStorage
---

# HDD Support

> In harvester v1.0, *StorageClass* is introduced. That also introduces the disk tag and node tag for it. With the above things, **HDD** is also supported with this version.

Reviewer: ```suggestion
Harvester v1.0 introduces *StorageClass* as well as the disk tags and node tags for it. With these features, Harvester v1.1 now supports **HDD**.
```

Reviewer: Does it mean harvester v1.1 now supports HDD?

Author: yes!  done for the above suggestions

:::note

HDD is not recommended for RKE2 clusters and regular use.

:::

> # Recommand Partice

Reviewer: ```suggestion
## Recommended Practices
```

> First, add your HDD on the host page. Giving suitable disk tags like `HDD` or `ColdStorage`. You can check [here](https://docs.harvesterhci.io/v1.1/host/#multi-disk-management) for more information if you do not understand how to add extra disk and disk tags.

Reviewer: ```suggestion
First, add your HDD on the `Host` page and specify the disk tags like `HDD` or `ColdStorage` as needed. For more information about how to add extra disks and disk tags, see [Multi-disk Management](https://docs.harvesterhci.io/v1.1/host/#multi-disk-management).
```

Author: done

> Then, create the new `StorageClass` for HDD (with the above disk tag). Remember that for some properties of HDD (large capability but slow performance), reducing the number of replicas may get better performance. Check [here](https://docs.harvesterhci.io/v1.1/advanced/storageclass) for more detail.

Reviewer: ```suggestion
Then, create a new `StorageClass` for the HDD (with the disk tag above). For HDDs that offer large capability but slow performance, you can reduce the number of replicas to improve the performance. For details, see [storageclass](https://docs.harvesterhci.io/v1.1/advanced/storageclass).
```

Author: done

> Now, you can create the volume with the above `StorageClass` that is used by HDD for cold storage or archive.

Reviewer: ```suggestion
Now, you can create a volume with the `StorageClass` used by the HDD for cold storage or archive.
```

Author: done
